cdap-app-fabric/src/test/java/co/cask/cdap/metadata/MetadataAdminAuthorizationTest.java

```diff
@@ -143,7 +143,12 @@ public void testSearch() throws Exception {
                      Collections.singleton(Action.EXECUTE));
 
     AppFabricTestHelper.deployApplication(Id.Namespace.DEFAULT, AllProgramsApp.class, "{}", cConf);
-    EnumSet<EntityTypeSimpleName> types = EnumSet.allOf(EntityTypeSimpleName.class);
+
+    // query for all metadata entity type except schedule
+    // TODO: https://issues.cask.co/browse/CDAP-14705, add back schedule type when the JIRA is fixed.
+    EnumSet<EntityTypeSimpleName> types = EnumSet.of(EntityTypeSimpleName.NAMESPACE, EntityTypeSimpleName.ARTIFACT,
```

Comment:
Because this will also need security extension change right, we will convert this Authorizable to SentryAuthorizable in the extension, and if we want to add any other entity id in the future, the search can break. So I think we should deal with it on a general purpose